src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@
 * Allow configurable gRPC channel options
   ([#1033](https://github.com/open-telemetry/opentelemetry-dotnet/pull/1033))
 
+* Rename extension method `AddOtlpExporter` to `AddOtlpExporter`.
```

Comment:
Looks like a copy/paste fail, should be `UseOtlpExporter` to `AddOtlpExporter` right?